haystack/nodes/prompt/invocation_layer/amazon_bedrock.py

```diff
@@ -258,7 +258,7 @@ class AmazonBedrockInvocationLayer(AWSBaseInvocationLayer):
         r"ai21.j2.*": AI21LabsJurassic2Adapter,
         r"cohere.command.*": CohereCommandAdapter,
         r"anthropic.claude.*": AnthropicClaudeAdapter,
-        r"meta.llama2.*": MetaLlama2ChatAdapter,
+        r"meta.llama.*": MetaLlama2ChatAdapter,
```

Comment:
Do we need this . here before *?

Reply:
Yes it's a regex match all char. Bit ugly because of the other dots, but we also added tests.